README.md

```diff
@@ -0,0 +1,61 @@
+# cloud-provider-aws
+The AWS cloud provider provides the interface between a Kubernetes cluster and AWS service APIs. This project allows a Kubernetes cluster to provision, monitor and remove resources necessary for operation of the cluster.
+
+## AWS Labels
+Before building out your cluster, you will need to make sure the your AWS resources are properly labeled. Label all k8s instances, your security group, vpc, and subnets with:
+
+KubernetesCluster = your_cluster_name
```

Comment:
This tag has been replaced by the owned/shared tag, see: https://github.com/kubernetes/kubernetes/issues/35052, https://github.com/kubernetes/kubernetes/issues/22058#issuecomment-308026369.  I actually don't know if I know all the cases where Kubernetes uses these tags.  Do you have any links to usages or documentation?